### PR TITLE
update connection pooling docs

### PR DESCRIPTION
**Pull request by vito**
closes #253

### CODE REVIEW

File: lit/docs/install/web.lit [hunk 1]
```diff
@@ -185,6 +176,31 @@ as performing all pipeline scheduling. It's basically the brain of Concourse.
...
+
+      The sum of these numbers across all \code{web} nodes should not be
+      greater than the maximum number of simultaneous connections your Postgres
+      server will allow. See \reference{db-resource-utilization}{\code{db} node
+      resource utilization} for more information.
+
+      For example, if 3 \code{web} nodes are configured as such:
+
+      \codeblock{bash}{{{
+      CONCOURSE_API_NAX_COONS=10     # default
```
Comment (cirocosta) [inline lit/docs/install/web.lit:197]:
```suggestion
      CONCOURSE_API_MAX_CONNS=10     # default
```

File: lit/docs/install/web.lit [hunk 2]
```diff
@@ -185,6 +176,31 @@ as performing all pipeline scheduling. It's basically the brain of Concourse.
...
+      CONCOURSE_BACKEND_NAX_COONS=50 # default
```
Comment (cirocosta) [inline lit/docs/install/web.lit:198]:
```suggestion
      CONCOURSE_BACKEND_MAX_CONNS=50 # default
```

Comment (pivotal-jamie-klassen) [inline lit/docs/install/web.lit:198]: oopsie
```suggestion
      CONCOURSE_API_MAX_CONNS=10     # default
      CONCOURSE_BACKEND_MAX_CONNS=50 # default
```

Comment (cirocosta) [reply lit/docs/install/web.lit:198]: you're late hehe

**Comment by vito** [comment]
haha, woops, fixed the typos

**Pull request merged**